e2e/testcases/correction-birth/correct-birth-record-3.spec.ts

```diff
@@ -0,0 +1,1200 @@
+      )
+    ).toBeVisible()
+
+    await expect(
+      page.getByText(
+        declaration.child.name[0].firstNames +
+          ' ' +
+          declaration.child.name[0].familyName
+      )
+    ).toBeVisible()
+    await expect(page.getByText('Verified')).toBeVisible()
+    await expect(
+      page.getByText(
+        'Informant did not provide this information (Material omission)'
+      )
+    ).toBeVisible()
+    await expect(
+      page.getByText(declaration.registration.registrationNumber)
+    ).toBeVisible()
+
+    await page.getByLabel('No').check()
+
+    /*
+     * Expected result: should enable the Send for approval button
+     */
+    await page.getByRole('button', { name: 'Send for approval' }).click()
+    await page.getByRole('button', { name: 'Confirm' }).click()
+
+    /*
+     * Expected result: should
+     * - be navigated to sent for approval tab
+     * - include the declaration in this tab
+     */
+    expect(page.url().includes('registration-home/approvals')).toBeTruthy()
+    await expect(page.locator('#navigation_outbox')).not.toContainText('1', {
+      timeout: 1000 * 30
+    })
+
+    await expect(
+      page.getByText(
+        declaration.child.name[0].firstNames +
+          ' ' +
+          declaration.child.name[0].familyName
+      )
+    ).toBeVisible()
+  })
+
+  test.describe('3.8 Correction Approval', async () => {
+    test.beforeAll(async ({ browser }) => {
+      await page.close()
+
+      page = await browser.newPage()
+
+      await login(page, 'k.mweene', 'test')
+      await createPIN(page)
+    })
+
+    test('3.8.1 Record audit by local registrar', async () => {
+      await page.getByPlaceholder('Search for a tracking ID').fill(trackingId)
+      await page.getByPlaceholder('Search for a tracking ID').press('Enter')
+      await page.locator('#ListItemAction-0-icon').click()
+      await page.getByRole('button', { name: 'Assign', exact: true }).click()
+
+      await page.locator('#name_0').click()
+    })
+
+    test('3.8.2 Correction review', async () => {
+      await page.getByRole('button', { name: 'Review', exact: true }).click()
+
+      /*
+       * Expected result: should show
+       * - Submitter
+       * - Requested by
+       * - Reason for request
+       * - Comments
+       * - Original vs correction
+       */
+
+      await expect(page.getByText('Submitter' + 'Felix Katongo')).toBeVisible()
+
+      await expect(
+        page.getByText(
+          'Requested by' +
+            declaration.child.name[0].firstNames +
+            ' ' +
+            declaration.child.name[0].familyName
+        )
+      ).toBeVisible()
+      await expect(
+        page.getByText(
+          'Reason for request' +
+            'Informant did not provide this information (Material omission)'
+        )
+      ).toBeVisible()
+
+      await expect(
+        page.getByText('Comments' + declaration.registration.registrationNumber)
+      ).toBeVisible()
+
+      /* assertion fails
```

Comment:
Is this something we are going to fix? I suggest adding @todo: to make it easier to find, or to remove